tools/tracegen/TraceGen.cpp

```diff
@@ -58,12 +58,13 @@ help(int argc, char *argv[])
 			|| 0 == strcmp(argv[i], "/h")
 			|| 0 == strcmp(argv[i], "/help")
 		) {
-			printf("%s [-threshold num] [-w2cd] [-generateCfiles] [-treatWarningAsError] [-root rootDir] [-force]\n", argv[0]);
+			printf("%s [-threshold num] [-w2cd] [-generateCfiles] [-treatWarningAsError] [-root rootDir] [-file file.tdf] [-force]\n", argv[0]);
```

Comment:
Depends on your definition of ok.  You probably wouldn't want to, but it will still work correctly